### PR TITLE
UGENE-6940. GUI tests for "Filter short scaffolds..." main menu action

### PR DESCRIPTION
**Pull request by evelinabis**

### CODE REVIEW

File: src/plugins/GUITestBase/GUITestBase.pro [hunk 1]
```diff
@@ -259,6 +259,7 @@ HEADERS +=  src/GUITestBasePlugin.h \
             src/tests/common_scenarios/workflow_designer/estimating/GTTestsWorkflowEstimating.h \
             src/tests/common_scenarios/workflow_designer/name_filter/GTTestsWorkflowNameFilter.h \
             src/tests/common_scenarios/workflow_designer/parameters_validation/GTTestsWorkflowParameterValidation.h \
+            src/tests/common_scenarios/workflow_designer/samples/GTTestsWorkflowSamples.h \
```
Comment (DmitriiSukhomlinov) [inline src/plugins/GUITestBase/GUITestBase.pro:262]: What kind of tests this file for? I'm not sure, that there is no way to add this test to an existing one, which is similar by purposes

**Review by DmitriiSukhomlinov — DISMISSED** [review]

File: src/plugins/GUITestBase/GUITestBase.pro (hunk 1 above)
Comment (evelinabis) [reply src/plugins/GUITestBase/GUITestBase.pro:262]: I don't know. I can move the test wherever you say
Probably this folder exists for tests 6940--6952

File: src/plugins/GUITestBase/src/GUITestBasePlugin.cpp [hunk 2]
```diff
@@ -3094,6 +3095,11 @@ void GUITestBasePlugin::registerTests(UGUITestBase *guiTestBase) {
     REGISTER_TEST(GUITest_common_scenarios_workflow_name_filter::test_0002);
     REGISTER_TEST(GUITest_common_scenarios_workflow_name_filter::test_0003);
 
+    /////////////////////////////////////////////////////////////////////////
+    // Common scenarios/Workflow designer/Samples
+    /////////////////////////////////////////////////////////////////////////
+    REGISTER_TEST_TL(GUITest_common_scenarios_workflow_samples::test_0001, minutes(7), labels({Cistrome, Linux, Windows, MacOS}));
```
Comment (yalgaer) [inline src/plugins/GUITestBase/src/GUITestBasePlugin.cpp]: The test itself looks simple, but you want to add to the special 'long-runing'  suite.
Unless we have a reason we should add tests to the Nightly (default) suite.
What is the execution time of the test?

**Review by yalgaer — CHANGES_REQUESTED** [review]

File: src/plugins/GUITestBase/src/GUITestBasePlugin.cpp (hunk 2 above)
Comment (evelinabis) [reply src/plugins/GUITestBase/src/GUITestBasePlugin.cpp]: 1 minute

**Review by yalgaer — APPROVED** [review]

**Pull request merged**